tests/browser-tests.js

```diff
@@ -0,0 +1,57 @@
+/* globals describe, before */
+
+// const expect = require('chai').expect
+const async = require('async')
+const store = require('local-storage-blob-store')
+const tests = require('./repo-test')
+const _ = require('lodash')
+const IPFSRepo = require('../src')
+
+global.Buffer = require('buffer').Buffer
```

Comment:
I'm not being able to find a way of doing this :( 

We need is `require('buffer').Buffer` and not just `require('buffer')`, and it seems that [`webpack.ProvidePlugin`](https://webpack.github.io/docs/shimming-modules.html#plugin-provideplugin) can't do it. 

Do you know any other way?